From 3850d22dfbc9f1838c58adfd8c041c24611bdaae Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Tue, 20 Dec 2011 02:42:27 +0200
Subject: [PATCH 0450/1092] usb: musb: cleanup kconfig

The whole thing depends on USB_MUSB_HDRC, just add an 'if'.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
Signed-off-by: Felipe Balbi <balbi@ti.com>
---
 drivers/usb/musb/Kconfig | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/drivers/usb/musb/Kconfig b/drivers/usb/musb/Kconfig
index f6e305f..f70cab3 100644
--- a/drivers/usb/musb/Kconfig
+++ b/drivers/usb/musb/Kconfig
@@ -30,9 +30,10 @@ config USB_MUSB_HDRC
 	  To compile this driver as a module, choose M here; the
 	  module will be called "musb-hdrc".
 
+if USB_MUSB_HDRC
+
 choice
 	prompt "Platform Glue Layer"
-	depends on USB_MUSB_HDRC
 
 config USB_MUSB_DAVINCI
 	tristate "DaVinci"
@@ -77,28 +78,24 @@ choice
 
 config USB_UX500_DMA
 	bool 'ST Ericsson U8500 and U5500'
-	depends on USB_MUSB_HDRC
 	depends on USB_MUSB_UX500
 	help
 	  Enable DMA transfers on UX500 platforms.
 
 config USB_INVENTRA_DMA
 	bool 'Inventra'
-	depends on USB_MUSB_HDRC
 	depends on USB_MUSB_OMAP2PLUS || USB_MUSB_BLACKFIN
 	help
 	  Enable DMA transfers using Mentor's engine.
 
 config USB_TI_CPPI_DMA
 	bool 'TI CPPI (Davinci)'
-	depends on USB_MUSB_HDRC
 	depends on USB_MUSB_DAVINCI
 	help
 	  Enable DMA transfers when TI CPPI DMA is available.
 
 config USB_TUSB_OMAP_DMA
 	bool 'TUSB 6010'
-	depends on USB_MUSB_HDRC
 	depends on USB_MUSB_TUSB6010
 	depends on ARCH_OMAP
 	help
@@ -106,7 +103,6 @@ config USB_TUSB_OMAP_DMA
 
 config MUSB_PIO_ONLY
 	bool 'Disable DMA (always use PIO)'
-	depends on USB_MUSB_HDRC
 	help
 	  All data is copied between memory and FIFO by the CPU.
 	  DMA controllers are ignored.
@@ -117,3 +113,5 @@ config MUSB_PIO_ONLY
 	  parameter.
 
 endchoice
+
+endif # USB_MUSB_HDRC
-- 
1.7.11.2

